integration_test/containertools.py

```diff
@@ -1,7 +1,8 @@
-import pytest
+import time
```

Comment:
Forgot to note this in the commit: line 45 calls `time.sleep`, but we had no import of `time`. Guess `make_stopped` never got to `inspect_container` fast enough to see the container not yet running.